app/experimenter/nimbus-ui/src/lib/visualization/utils.tsx

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { DISPLAY_TYPE, METRIC, TABLE_LABEL } from "./constants";
+
+export const getTableDisplayType = (
+  metricKey: string,
+  tableLabel: string,
+  isControl: boolean,
+): DISPLAY_TYPE => {
+  let displayType;
+  switch (metricKey) {
+    // case METRIC.USER_COUNT:
+    //   displayType = DISPLAY_TYPE.POPULATION;
+    //   break;
```

Comment:
keep/remove?

Reply:
Keeping, we will need this in another table.